### PR TITLE
Fixes #9648: Add a node to the rudder-vagrant platform

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9648

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: repo.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+KEY=474A19E8
+
+# Make sure we don't run interactive commands
+export DEBIAN_FRONTEND=noninteractive
+
+# Accept the Rudder repository key
+wget --quiet -O- "https://${KEYSERVER}/pks/lookup?op=get&search=0x${KEY}" | apt-key add -
+
+# Add latest Rudder repository
+echo "deb http://www.rudder-project.org/apt-latest/ trusty main" > /etc/apt/sources.list.d/rudder.list
```
Comment (ncharles) [inline repo.sh]: could it be 
echo "deb http://www.rudder-project.org/apt-latest/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
like in the doc ?
(and ensure that lsb_release has been installed before

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Tested with 4.0rc5 too.

**Comment by amousset** [comment]
@ncharles Added lsb_release.

**Pull request merged**